Resources/template.json

```diff
@@ -168,6 +186,32 @@
     "deployAse": "[not(equals(parameters('aseDomainName'), ''))]"
   },
   "resources": [
+    {
+      "type": "Microsoft.Resources/deployments",
+      "name": "sharedManagedInstanceEnvironment",
+      "apiVersion": "2017-05-10",

```

Comment:
Assuming the vnet is only for DEV/SIT envs, should this not have "condition": "[equals(parameters('envType'),'dev')]", line added